.github/containerscan/allowedlist.yaml

```diff
@@ -3,10 +3,13 @@ general:
     # list of CVEs that are currently unfixed

```

Comment:
Please revert these changes as these has been taken care of

Reply:
Reverted the changes